pyomo/contrib/sensitivity_toolbox/tests/test_sens_sensitivity_calculation.py

```diff
@@ -0,0 +1,743 @@
+# ____________________________________________________________________________
+#
+# Pyomo: Python Optimization Modeling Objects
+# Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+# Under the terms of Contract DE-NA0003525 with National Technology and
+# Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+# rights in this software.
+# This software is distributed under the 3-clause BSD License.
+# ____________________________________________________________________________
+
+"""
+Unit Tests for interfacing with sIPOPT and k_aug
+"""
```

Comment:
Is this file a copy of `test_sens.py` that was added by mistake?

Reply:
I changed the line to 'Unit Tests for interfacing with sensitivity_calculation'. `test_sens_sensitivity_calculation.py` tests the sensitivity_calculation interface.